.github/workflows/ci-nochanges.yml

```diff
@@ -1,7 +1,7 @@
-name: CI
+name: CI No Changes
```

Comment:
I don't think we can change this. We need the name to match exactly. "CI" is a required status check, so we need a job with that name to always pass, whether or not there are meaningful changes.